kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -550,6 +571,14 @@ define([
             return this.ui.getElement(LOG_LINES).lastChild;
         }
 
+        doScrollToTop() {
```

Comment:
extracted out functions for scrolling to the top or the bottom of the log content panel